packages/backend/src/index.ts

```diff
@@ -35,7 +34,6 @@ import { PluginEnvironment } from './types';
 
 function makeCreateEnv(config: Config) {
   const root = getRootLogger();
-  const reader = UrlReaders.default({ logger: root, config });
```

Comment:
seems like this is no longer in the backend-common package. anyway this is all going to be dropped in the next change when we move to the new backend system. i can still load the preview environment without this part of the plugin env